broker/fragment/store_azure.go

```diff
@@ -29,30 +35,63 @@ type azureBackend struct {
 	endpoint    string
 	accountName string
 	client      pipeline.Pipeline
-	credentials *azblob.SharedKeyCredential
+	svcClient   service.Client
 	clientMu    sync.Mutex
+	udc         *service.UserDelegationCredential
+	udc_exp     *time.Time
 }
 
 func (a *azureBackend) Provider() string {
 	return "azure"
 }
 
+// Cache UserDelegationCredentials and refresh them when needed
+func (a *azureBackend) GetUserDelegationCredential() (*service.UserDelegationCredential, error) {
+	if a.udc == nil || (a.udc_exp != nil && a.udc_exp.After(time.Now().Add(time.Minute*10))) {
+		expTime := time.Now().Add(time.Hour * 24)
```

Comment:
nit: please see the other comment I wrote about code style in this repo being `var variableName =`

Reply:
Ok sorry... is that for all variables? I'm still not super familiar with these patterns in Go, but I'm seeing a bunch of different variable declaration styles in this file from before I made any changes and was just following along with what I saw.

This one seems to be using `:=`: 
https://github.com/gazette/core/blob/fc3ba772a67798314b8fe54f1e4d2976e4c527f0/broker/fragment/store_azure.go#L41

This one is using `var` but not actually assigning a value until later:
https://github.com/gazette/core/blob/fc3ba772a67798314b8fe54f1e4d2976e4c527f0/broker/fragment/store_azure.go#L110-L118

This one taught me something new: apparently return values can be named, and those are treated as implicit declarations:
https://github.com/gazette/core/blob/fc3ba772a67798314b8fe54f1e4d2976e4c527f0/broker/fragment/store_azure.go#L170

Would you like me to change everything here to using `var`? Or is there a linter I'm missing or something?